shell.nix

```diff
@@ -31,6 +35,11 @@ in
     [ -n "$1" ] || { printf "Usage: wrapFactor <factor-root>" ; return; }
     local root="$(realpath $1)"
     local binary="''${root}/factor"
+    local wrapped="''${root}/.factor-wrapped"
+    # Remove the wrapped binary if a new VM has been compiled
+    ${lib.getBin file}/bin/file $binary |grep ELF >/dev/null && rm -f "$wrapped"
```

Comment:
does ``$binary`` need to be in quotes in case ``root`` has spaces in it?

Reply:
I think you are correct.